config/initializers/md5_replace.rb

```diff
@@ -0,0 +1,6 @@
+# TODO: configure ActiveSupport::Digest after Rails 5.2 update
+
+raise "setup ActiveSupport::Digest on Rails 5.2+" if Gem::Version.new(Rails.version) >= Gem::Version.new('5.2')
+
+::Digest.send :remove_const, :MD5
+::Digest::MD5 = ::Digest::SHA256
```

Comment:
Monkey patches like this make me cringe for the simple fact of confusion.  Now the `MD5` class will actually perform a `SHA256`.  Since this is all throwaway code though, I think it's ok, but instead of a `TODO` I'd use a `FIXME`.  `TODO` tells me an optimization or feature might be worth implementing in the future where a `FIXME` tells me this is definitely not the right way, but we are ok with it for now.

Reply:
We will not do this. Too risky of bad confusion provided so many MD5 references in other gems.